pkg/provision/providers/qemu/node.go

```diff
@@ -127,10 +141,16 @@ func (p *provisioner) createNode(state *vm.State, clusterReq provision.ClusterRe
 
 	// no need to wait here, as cmd has all the Stdin/out/err via *os.File
 
+	nodeType := machine.Type(-1) // HACK!
```

Comment:
oh I probably want to fix that before we merge it, I think I saw `TypeUnknown` in one of the PRs